### ISSUE TITLE
Screen flickers black badly with all games/emulators when using certain frame buffer options with Discord Screen share

### ISSUE THREAD

**Issue by Blazingflare** [gonetz/GLideN64#2283]
I ran into a pretty major problem that only happens under very specific circumstances to every game tested and may be hardware specific.
I usually don't have any problems with any game, but if I have "Use emulator help to read/write framebuffers" checked and turn on Discord's game capture (but not the window screen share), the video in the emulator starts flickering black pretty intensely. This happens with all versions of PJ64 I tested including the latest, all versions of Bizhawk tested including the latest, and the latest build of Mupen64. And it happens with version 3.0, 4.0, and the latest build of GLideN64.
If I uncheck the option, everything works fine as expected again. 

My CPU is a AMD Ryzen 5 2600X Six-Core Processor 
And my GPU is a NVIDIA GeForce GTX 970

My PC is more than powerful enough to run the slightly more demanding settings and stream with plenty of power to spare, so it really isn't much of an issue. And the same issue doesn't happen with other streaming software meaning that Discord is partly to blame too. But it's still an issue with the default settings.

**Comment by IronYoshi64** [comment]
Yeah I have a feeling this is specifically a GLideN64 problem. Discord does not seem to have a problem with Jabo Direct3D, Glide64/Project64 Video, or angrylion, so I think it really is just a GLideN64 issue.

Although that can be a workaround, its not perfect. You need to have game already open, then start streaming it through Discord. If the screen flickers, simply try a Reset of the game, then for me, that fixed it every time. If that doesn't fix it the first time, stop the stream, then do it again. :smile:
Also going fullscreen is not recommended. This causes the flickering to return and didn't go away until another Reset, so play your game in Windowed mode.

Also I found out, that option may not have to be off? I did this same things of what I said above and it still works? So i think as a workaround for this, you need to have the game open, then stream it through Discord, then Reset game and there you go. 

Although this is a workaround, I feel like it will be good if this issue was truly fixed sometime.

**Comment by CadetSparklez** [comment]
to add on to this, playing rogue squadron in 1964 and either sharing my screen or recording it with obs (either display, game or window capture) causes some strange artifacts to happen periodically, ill have to see if they show up in recordings so I can send a video, all also get a friend to let me know if they show up in the screen shares

**Comment by IronYoshi64** [comment]
I find that if you just turn on the Overscan option, this fixes the discord streaming issue stuff.
I posted this a while ago as well: https://github.com/gonetz/GLideN64/issues/2508

**Comment by Squall-Leonhart** [comment]
discord interfers with opengl texture binding, it may be required to rebind every frame when discord is hooked.
